src/publish.proj

```diff
@@ -22,9 +50,24 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="UploadTestsToAzure">
+    <ItemGroup>
+      <TestArchives>
```

Comment:
Please comment that you are adding additional metadata here to all `TestArchives` items.

Reply:
Moved to initial declaration of Items